src/store/cart.js

```diff
@@ -1,18 +1,27 @@
 import axios from 'axios';
 
-const cart = (state = { lineItems: [] }, action) => {
-  if (action.type === 'SET_CART') {
+const cart = (state = { lineItems: [] }, action)=> {
+  if(action.type === 'SET_CART' || action.type === 'REMOVE_ITEM' || action.type === 'ADD_ITEM' || action.type === 'CREATE_ORDER'){
     return action.cart;
   }
-  if (action.type === 'ADD_ITEM' || action.type === 'REMOVE_ITEM') {
-    return { ...state, lineItems: action.cart.lineItems.sort((a,b) => a.createdAt.localeCompare(b.createdAt)) }
```

Comment:
we need these lines to sort the items in the cart by the time they were added: i.e. from first added to last added